backbone.layoutmanager.js

```diff
@@ -86,9 +86,9 @@ var LayoutManager = Backbone.View.extend({
     if (beforeRender) {
       var ret = beforeRender.call(view, view);
 
-      if (ret && ret.then) {
+      if (ret && ret.promise) {
```

Comment:
This specific change might not be good looking at it again.  Before render returns a promise and not a view, right?